tests/sensors/test_timeout_sensor.py

```diff
@@ -72,6 +74,7 @@ def setUp(self):
         }
         self.dag = DAG(TEST_DAG_ID, default_args=args)
 
+    @pytest.mark.quarantined
```

Comment:
Have we seen this test fail? I'm surprised by that since it appears to be using freezegun for mock time.

Reply:
Yes. I saw it failed. For now I want to make all the "suspicious" cases and possibly have some false negatives". The idea is that I do not want to do all the investigation myself but split it between committers so that everyone will take care about some of the "flaky" tests, observe it's stability and move it out of quarantine when it is stable (and move it back if not). I think all or most committers should take part of the job on their shoulders if we wan to keep stability for everyone.